api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/Claim.java

```diff
@@ -84,4 +93,11 @@ public void updateEligibilityStatus(EligibilityStatus newStatus) {
         }
         eligibilityStatus = newStatus;
     }
+
+    public void updateCardStatus(CardStatus newStatus) {
```

Comment:
We may need to invoke this method from within updateClaimStatus - if claimStatus is set to pending expiry or expired then we set the card status to pending_cancellation (unless it is already to be cancelled or cancelled). We might need a bit of a state machine...
Also if a claim goes back to active we need to ensure that card_status is set to active. Admittedly all this might be better handled in another PR.